ods_ci/tests/Tests/0100__platform/0101__deploy/0104__operators/0104__rhods_operator/0110__service_mesh.robot

```diff
@@ -58,6 +61,28 @@ Validate Service Mesh State Removed
     [Teardown]    Set Service Mesh State To Managed And Wait For CR Ready
     ...           ${SERVICE_MESH_CR_NAME}    ${SERVICE_MESH_CR_NS}    ${OPERATOR_NS}
 
+Validate Service Mesh Control Plane Already Created
+    [Documentation]    This Test Case validates that only one ServiceMeshControlPlane is allowed to be installed per project/namespace
+    [Tags]      Operator        Tier3       RHOAIENG-2517
+    Fetch Image Url And Update Channel
+    Fetch Cluster Type By Domain
+    IF    "${CLUSTER_TYPE}" == "selfmanaged"
+        Uninstall RHODS In Self Managed Cluster
+        Create Smcp From Template
+        Install RHODS In Self Managed Cluster Using CLI     ${CLUSTER_TYPE}     ${IMAGE_URL}
+    ELSE IF    "${CLUSTER_TYPE}" == "managed"
+        Uninstall RHODS In OSD
+        Create Smcp From Template
+        Install RHODS In Managed Cluster Using CLI      ${CLUSTER_TYPE}     ${IMAGE_URL}
+    END
+    Operator Deployment Should Be Ready
+    # Go check the Operator logs for the error message: denied the request: only one service mesh may be installed per project/namespace
+    Verify Pod Logs Contain Error
+    # Cleanup the SMCP
+    Delete Smcp
+    # Cleanup Olminstall dir
+    Cleanup Olm Install Dir
+
```

Comment:
We need a teardown method for the test just in case some step fails, so we are back to initial state, with ODH/RHOAI installed and working

Reply:
There is the suite teardown 
<img width="383" alt="image" src="https://github.com/user-attachments/assets/d3f3306f-872a-4655-a467-fb6c6b8698df">

Reply:
I agree with Cesar, we need a teardown which restore the correct RHOAI status in the cluster

Reply:
Ok I can move it to the Suite Teardown

Reply:
Added Test Case Teardown